src/app/services/teachers.ts

```diff
@@ -0,0 +1,11 @@
+export interface Iteachers {
+  avatar: "string";
+  dateOfBirth: "string";
+  email: "string";
+  firstname: "string";
+  id: 0;
```

Comment:
why do you predefine id?
it is must be not required

Reply:
this format from example value 
https://fierce-shore-32592.herokuapp.com/swagger-ui.html#!/teacher-controller/getallUsingGET